distributed-coordinator/src/main/java/com/gentics/mesh/distributed/coordinator/MasterElector.java

```diff
@@ -302,7 +306,7 @@ private void giveUpMasterFlag() {
 	 * @return true for the master
 	 */
 	private static boolean isMaster(Member member) {
-		return member.getBooleanAttribute(MASTER) == Boolean.TRUE;
+		return Boolean.parseBoolean(member.getAttributes().getOrDefault(MASTER, "false")) == Boolean.TRUE;
```

Comment:
What I meant was: the member attributes can only be set when a member (meaning: a mesh instance in the cluster) is created, but they never can be changed in runtime. This means it is no longer possible to "mark" a cluster member as being the current master by changing the member attributes (like it was possible in hazelcast 3).
So this check does not make any sense, because this will always return "false" for all members.
In the method electMaster() the elected master is marked by setting the flag in its MeshMemberInfo instance, so the check whether a specific member is the current master should read this flag, shouldn't it?